### PR TITLE
Deprecate TimeToLivePropertyPath

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Deprecate TimeToLivePropertyPath

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/Settings/ContainerProperties.cs [hunk 1]
```diff
@@ -262,6 +262,9 @@ public string PartitionKeyPath
         /// for a item is decided based on the value of this property in item.
         /// By default, TimeToLivePropertyPath is set to null meaning the time to live is based on the _ts property in item.
         /// </value>
+#if !INTERNAL
+        [Obsolete]
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/ContainerProperties.cs]: [Obsolete] has an optional parameter which is a text, that we can use to explain the alternative to this obsolete method.

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/ContainerProperties.cs]: Its full feature deprecation.

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosContainerTests.cs [hunk 2]
```diff
@@ -589,6 +589,7 @@ public async Task ImplicitConversion()
         /// This test verifies that we are able to set the ttl property path correctly using SDK.
         /// Also this test will successfully read active item based on its TimeToLivePropertyPath value.
         /// </summary>
+        [Obsolete]
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosContainerTests.cs]: Context text on why it get Obsolete would be good to have

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosContainerTests.cs]: Its a feature deprecation. Its primarily meant for internal usage and got leaked out.

**Pull request merged**